app/lib/app/modules/home/home_page.dart

```diff
@@ -12,16 +15,119 @@ class HomePage extends StatefulWidget {
 
 class _HomePageState extends ModularState<HomePage, HomeController> {
   //use 'controller' variable to access controller
+  bool isNight = false;
+  bool isBookmark = false;
+  String weatherMainAnimation = "clouds_night";
+  String weatherMain = "Clouds";
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text(widget.title),
       ),
-      body: Column(
-        children: <Widget>[],
+      backgroundColor: Colors.white,
+      body: ListView(
+        padding: const EdgeInsets.all(32),
+        children: <Widget>[
+          _buildCheckBoxNight(),
+          _buildCheckBoxBookmark(),
+          _builDropDown(),
+          Center(
+            child: Padding(
+              padding: const EdgeInsets.all(16),
+              child: Text("$weatherMain selecionado!!"),
+            ),
+          ),
+        ],
       ),
+      floatingActionButton: FloatingActionButton(
+        child: Text("ABRIR"),
```

Comment:
Hi @pedrohsampaioo.

Going to change to "**OPEN**"?

Tks

Reply:
Hi, sure! 😆